ai-platform-unified/notebooks/official/pipelines/control_flow_kfp.ipynb

```diff
@@ -1,820 +1,1037 @@
 {
```

Comment:
<p>You might add something to the effect of: 'you only need to run these once per service account'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>added</p>